certified-connectors/Cognizant Automation Center/Readme.md

```diff
@@ -0,0 +1,28 @@
+# Cognizant Automation Center Connector
+Cognizant Automation Center I&O is an integrated portfolio of Bots, services IPs, platform and solutions that brings the best of the breed Automated solutions. It is an AIOps platform that integrates with several systems in a complex IT ecosystem and facilitates transition Eliminate-Automation–Modernize using Devops SRE Operate framework. Cognizant Automation Center Integrates systems of engagement, systems of record, systems of monitoring & assurance and systems that execute Automation actions.
+

```

Comment:
@SelvaGanesh-Cognizant Please refer the [template provided for readme.md](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) file and add required sections.